src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.cs

```diff
@@ -874,28 +937,5 @@ protected override ParseOptions OptionsCore
             ImmutableDictionary<string, ReportDiagnostic>? diagnosticOptions)
             => Create(root, options, path, encoding, diagnosticOptions, isGeneratedCode: null);
 
-        // 2.8 BACK COMPAT OVERLOAD -- DO NOT MODIFY
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public static SyntaxTree ParseText(
```

Comment:
I don't believe we can remove this one.

Reply:
It's just moving up in the file since it's not a back compat overload anymore.

Reply:
I do not see an addition above the corresponds to this removal.

---
In reply to: [460243660](https://github.com/dotnet/roslyn/pull/44331#discussion_r460243660) [](ancestors = 460243660)

Reply:
AFAICT, the only addition has an optional `Encoding` parameter. If the public API analyzer isn't flagging this, that's very concerning.

---
In reply to: [460286910](https://github.com/dotnet/roslyn/pull/44331#discussion_r460286910) [](ancestors = 460286910,460243660)

Reply:
I found it. However, that API should not be added, and we should leave this as is.

---
In reply to: [460287155](https://github.com/dotnet/roslyn/pull/44331#discussion_r460287155) [](ancestors = 460287155,460286910,460243660)